types/react-fa/index.d.ts

```diff
@@ -37,15 +37,7 @@ export const Icon: ComponentClass<IconProps>;
 
 export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
     size?: IconSize;
-
-    // Does not work with TypeScript 2.3:
-    // Type '{ size: "lg"; children: Element[]; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<IconStackProps, ComponentState>> & Reado...'.
-    //   Type '{ size: "lg"; children: Element[]; }' is not assignable to type 'Readonly<IconStackProps>'.
-    //     Types of property 'children' are incompatible.
-    //       Type 'Element[]' is not assignable to type 'IconProps[]'.
-    //         Type 'Element' is not assignable to type 'IconProps'.
-    //           Property 'name' is missing in type 'Element'.
-    // children?: IconProps[];
+    children?: JSX.Element[];  // All JSX expression has type JSX.Element
```

Comment:
also note, that this allows a stack which is not really a stack (which is technically supported but not really something anyone would do). If there is desire to prevent that through typings, we can remove the non-array variant from the type union.